services/codeclimate/codeclimate-coverage.tester.js

```diff
@@ -46,7 +46,7 @@ t.create('test coverage when outer user repos query returns multiple items')
               },
               latest_default_branch_test_report: {
                 data: {
-                  id: '6463c8a3e3bc340001004bce',
+                  id: '65a1662cb0077b00013cb4de',
```

Comment:
I feel like this is not a sustainable solution for this test, but I think I'm not going to deal with re-writing this test in this PR as it is a bit of an yak-shave that isn't really related to the original point of this PR.